src/main.js

```diff
@@ -414,10 +418,32 @@ function handleAppWebContentsCreated(dc, contents) {
   });
 
   contents.on('new-window', (event, url) => {
-    if (isTrustedURL(url)) {
+    event.preventDefault();
```

Comment:
This is what was initially preventing the Jira v2 connect popup window from opening.

The subsequent code checks if the url is trusted and if so, opens up an Electron Browser window and loads the requested url into it.